lib/features/on_boarding/domain/repositories/on_boarding_repository.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:dartz/dartz.dart';
+
+import '../../../../core/failures.dart';
+
+abstract class OnBoardingRepository {
+  Future<Option<Failure>> createUser(String token, String storeName, String phoneNumber);
+}
```

Comment:
add comma and format after `phoneNumber`
```suggestion
abstract class OnBoardingRepository {
  Future<Option<Failure>> createUser(String token, String storeName, String phoneNumber,);
}
```